molgenis-data-security/src/main/java/org/molgenis/data/security/auth/UserMetadata.java

```diff
@@ -75,7 +75,8 @@ public void init() {
         .setLabel("Username")
         .setUnique(true)
         .setNillable(false)
-        .setReadOnly(true);
+        .setReadOnly(true)
+        .setValidationExpression("$('" + USERNAME + "').matches(/^[\\S].+[\\S]$/).value()");
```

Comment:
This pattern does allow spaces in the middle of the username. Should that be possible?